src/Microsoft.Data.SqlClient/netcore/src/Interop/SNINativeMethodWrapper.Windows.cs

```diff
@@ -34,100 +36,106 @@ internal static int SniMaxComposedSpnLength
 
         #region DLL Imports
+        
+        internal static uint SNIAddProvider(SNIHandle pConn, Provider ProvNum, [In] ref uint pInfo) =>
+            NativeMethods.SniAddProvider(pConn, ProvNum, ref pInfo);
+        
+        internal static uint SNIAddProvider(SNIHandle pConn, Provider ProvNum, [In] ref AuthProviderInfo pInfo) =>
+            NativeMethods.SniAddProvider(pConn, ProvNum, ref pInfo);
+        
+        internal static uint SNICheckConnection([In] SNIHandle pConn) =>
+            NativeMethods.SniCheckConnection(pConn);
+        
+        internal static uint SNIClose(IntPtr pConn) =>
+            NativeMethods.SniClose(pConn);
+        
+        internal static void SNIGetLastError(out SniError pErrorStruct) =>
+            NativeMethods.SniGetLastError(out pErrorStruct);
+        
+        internal static void SNIPacketRelease(IntPtr pPacket) =>
+            NativeMethods.SniPacketRelease(pPacket);
+        
+        internal static void SNIPacketReset([In] SNIHandle pConn, IoType IOType, SNIPacket pPacket, ConsumerNumber ConsNum) =>
+            NativeMethods.SniPacketReset(pConn, IOType, pPacket, ConsNum);
+        
+        internal static uint SNIQueryInfo(QueryType QType, ref uint pbQInfo) =>
+            NativeMethods.SniQueryInfo(QType, ref pbQInfo);
+        
+        internal static uint SNIQueryInfo(QueryType QType, ref IntPtr pbQInfo) =>
+            NativeMethods.SniQueryInfo(QType, ref pbQInfo);
+        
+        internal static uint SNIReadAsync(SNIHandle pConn, ref IntPtr ppNewPacket) =>
+            NativeMethods.SniReadAsync(pConn, ref ppNewPacket);
+        
+        internal static uint SNIReadSyncOverAsync(SNIHandle pConn, ref IntPtr ppNewPacket, int timeout) =>
+            NativeMethods.SniReadSyncOverAsync(pConn, ref ppNewPacket, timeout);
+        
+        internal static uint SNIRemoveProvider(SNIHandle pConn, Provider ProvNum) =>
+            NativeMethods.SniRemoveProvider(pConn, ProvNum);
+        
+        internal static uint SNISecInitPackage(ref uint pcbMaxToken) =>
+            NativeMethods.SniSecInitPackage(ref pcbMaxToken);
+        
+        internal static uint SNISetInfo(SNIHandle pConn, QueryType QType, [In] ref uint pbQInfo) =>
+            NativeMethods.SniSetInfo(pConn, QType, ref pbQInfo);
+        
+        internal static uint SNITerminate() =>
+            NativeMethods.SniTerminate();
+        
+        internal static uint SNIWaitForSSLHandshakeToComplete([In] SNIHandle pConn, int dwMilliseconds, out uint pProtocolVersion) =>
+            NativeMethods.SniWaitForSslHandshakeToComplete(pConn, dwMilliseconds, out pProtocolVersion);
+        
+        internal static uint UnmanagedIsTokenRestricted([In] IntPtr token, [MarshalAs(UnmanagedType.Bool)] out bool isRestricted) =>
+            NativeMethods.SniIsTokenRestricted(token, out isRestricted);
+        
+        private static uint GetSniMaxComposedSpnLength() =>
+            NativeMethods.SniGetMaxComposedSpnLength();
```

Comment:
This kind of situation has around a dozen `private static` methods which are only each called 1-3 times within SNINativeMethodWrapper, and which do nothing but call their corresponding methods on the `NativeMethods` variable. We could just as easily discard them and update the call sites to call `NativeMethods` directly.

Reply:
100% agree - these are all removed in a cleanup PR. I wanted to limit the amount of changes going into this one so the later PR(s) make sense https://github.com/benrr101/SqlClient/commit/1cc5f16d8fe7018494b7db31593804427f49dc83